freezeyt/freezing.py

```diff
@@ -1,5 +1,7 @@
 from urllib.parse import urlparse, urljoin
 from pathlib import Path
+from werkzeug.datastructures import TypeConversionDict
```

Comment:
Why do you use `TypeConversionDict` here?
We should always get strings from WSGI.

Reply:
I supposed that is better to use werkzeug to convert to dict when you advice to look at werkzeug. I think that is better to convert list of tuples to dict because it is more readable and simplier than use a for cycle over list. werkzeug give us conversion dict for http cases etc. so I supposed that be better convert header as a part of http response by specific dict instead of python dict. But we still work with strings so at first view theres is no diff. But I dont know if there is not something behind.

Reply:
Ah! Sorry, I meant something like [parse_options_header](https://werkzeug.palletsprojects.com/en/1.0.x/http/#werkzeug.http.parse_options_header) rather than `split('; ')` for the Content-Type.
The reason is that, for example, the [standard](https://tools.ietf.org/html/rfc7231#section-3.1.1.5) says there [might or might not](https://tools.ietf.org/html/rfc7231#section-3.1.1.1) be a space after the `;`. The library should get details like this right.
For `TypeConversionDict`, this is not necessary. AFAIK, werkzeug mostly uses this to handle numeric data (like `Content-Length: 1234`, where if it's not a number, it's like the header was missing). And to maybe to make extra sure it's using the right datatype. But our headers come from a WSGI app. The types are well defined; if a WSGI app sends non-string in the headers, it's their fault :)


Anyway, when reading the RFC, I noticed that the content-type is case-insensitive. Could you please disregard case when comparing it, an add a test for e.g. `TEXT/HTML`?

Reply:
> is better to convert list of tuples to dict because it is more readable and simplier than use a for cycle over list

And I do agree with that! Just consider a [different dict type](https://werkzeug.palletsprojects.com/en/1.0.x/datastructures/?highlight=multidict#werkzeug.datastructures.Headers) :)

Reply:
OK, thank you.